scene/gui/code_edit.cpp

```diff
@@ -732,15 +732,19 @@ void CodeEdit::_backspace_internal(int p_caret) {
 			if (idx != -1) {
 				prev_column = cc - auto_brace_completion_pairs[idx].open_key.length();
 
+				int new_cc;
 				if (_get_auto_brace_pair_close_at_pos(cl, cc) == idx) {
-					remove_text(prev_line, prev_column, cl, cc + auto_brace_completion_pairs[idx].close_key.length());
+					new_cc = cc + auto_brace_completion_pairs[idx].close_key.length();
 				} else {
-					remove_text(prev_line, prev_column, cl, cc);
+					new_cc = cc;
 				}
```

Comment:
Could be simplified to:
```cpp
int new_cc = cc;
if (_get_auto_brace_pair_close_at_pos(cl, cc) == idx) {
	new_cc += auto_brace_completion_pairs[idx].close_key.length();
}
```

Reply:
`cc` is still used further down in other branches and iterations of the loop, so modifying it might change behavior. Does it fix further bugs by doing so or actually risk introducing bugs?

Reply:
There is a `continue` after this, and it is re-set on the next loop iteration, so it doesn't affect any other uses.

Reply:
Ah indeed :+1: